src/patternfly/components/Table/table-th.hbs

```diff
@@ -25,8 +25,14 @@
     {{{table-th--attribute}}}
   {{/if}}>
   {{#if table-th--sortable}}
-    {{#> button button--modifier="pf-m-plain"}}
-      {{> @partial-block}}
+    <button class="pf-c-button pf-m-pain{{#if table-th--sortable-wrap}} pf-m-sort-wrap{{/if}}">
+      {{#if table-th--sortable-wrap}}
```

Comment:
@mcoker @mattnolting what do you think about wrapping the text in a span only if `pf-m-sort-wrap` is applied? This was mainly so that we could target the span to apply `white-space: normal` so that only the text wraps. Or we could have a span wrap all text that is inside sortable `<th>`